### PR TITLE
Convert FORM_LINK_WORKFLOW FF to a Standard plan privilege

### PR DESCRIPTION
**Pull request by sravfeyn**
## Product Description

This moves the FORM_LINK_WORKFLOW toggle to a Standard plan privilege within the existing End of Form Navigation feature.
https://dimagi-dev.atlassian.net/browse/SC-2410

## Feature Flag
Removed FORM_LINK_WORKFLOW toggle

### Safety story
I have tested this locally for apps built with toggle enabled previously before removing the toggle and I have tested that the feature works without the toggle for fresh apps.

### Automated test coverage
Tests exist

### QA Plan

NA, I have done UI tests myself as they are minimal

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions

If the toggle deletion needs to be reverted we need to identify domains that start using the feature without the toggle and then enable the toggle for those domains after reverting

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Comment by sravfeyn** [comment]
This needs a bit more work on the strategy for dealing with domains that are below standard plan and have enabled the flag. Currently I have [captured](https://docs.google.com/spreadsheets/d/1gElrI1CJPxzsQPw4A-u-9HV3SDwpHEgHZB2_iKrpsus/edit#gid=0) the subscription data for all the domains that have the flag enabled.

**Comment by sravfeyn** [comment]
@snopoke  @biyeun @dannyroberts 

Just checking for others inputs on how to deal with domains that are under a plan that we want to upgrade the feature flag to. This is likely going to be an issue with other flags that will be GAed soon, so arriving at a good solution will make it easy for others to follow. I am happy to implement those, but want to get input from the wider team.


Graham suggested taking the approach of creating a custom plan using a [method](https://dimagi.slack.com/archives/C02QG63MU/p1671521184163299) he followed earlier. This is likely the best approach we have so far, but I am not sure if this will create too many accounting data models when we do this for large number of feature flags.

**Comment by snopoke** [comment]
> @snopoke @biyeun @dannyroberts
> 
> Just checking for others inputs on how to deal with domains that are under a plan that we want to upgrade the feature flag to. This is likely going to be an issue with other flags that will be GAed soon, so arriving at a good solution will make it easy for others to follow. I am happy to implement those, but want to get input from the wider team.
> 
> @gherceg suggested taking the approach of creating a custom plan using a [method](https://dimagi.slack.com/archives/C02QG63MU/p1671521184163299) he followed earlier. This is likely the best approach we have so far, but I suspect if this will create too many accounting data models when we do this for large number of feature flags.

@sravfeyn I'm not sure what the best approach here is. This seems like something you should discuss with the SaaS ops team?

**Comment by sravfeyn** [comment]
I have added a CEP https://github.com/dimagi/commcare-hq/issues/32482 that I am planning to implement to deal with the problem of enabling for domains that may not have paid plan.

File: corehq/apps/accounting/migrations/0064_add_form_link_workflow_priv.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0063_replace_linked_projects_ff_with_erm'),
+    ]
+
+    operations = [
+        migrations.RunPython(_grandfather_form_link_workflow_privs),
```
Comment (SmittieC) [inline corehq/apps/accounting/migrations/0064_add_form_link_workflow_priv.py:28]: If I'm not mistaken, this migration is not revertable if a "down" method is not specified. That is if we want it to be revertable

Comment (sravfeyn) [reply corehq/apps/accounting/migrations/0064_add_form_link_workflow_priv.py:28]: We don't usually add a down method for these sort of migrations,

File: corehq/toggles/__init__.py [hunk 2]
```diff
@@ -537,7 +537,7 @@ def all_toggles_by_name_in_scope(scope_dict, toggle_class=StaticToggle):
     result = {}
     for toggle_name, toggle in scope_dict.items():
         if not toggle_name.startswith('__'):
-            if isinstance(toggle, toggle_class):
+            if type(toggle) == toggle_class:
```
Comment (snopoke) [inline corehq/toggles/__init__.py:540]: nit: add comment to indicate why `isinstance` can't be used

Comment (sravfeyn) [reply corehq/toggles/__init__.py]: Added

**Comment by sravfeyn** [comment]
Build is passing now. Bumping to see if there is any further feedback on this PR.

**Comment by sravfeyn** [comment]
Hey @snopoke does this look good to you now?

**Comment by sravfeyn** [comment]
@snopoke

**Review by snopoke — APPROVED** [review]

**Pull request merged**